draft-ietf-dnsop-avoid-fragmentation.mkd

```diff
@@ -429,6 +447,10 @@ default path MTU size and requestor's maximum UDP payload size.
   their measurements suggest setting the EDNS0 requestor's UDP payload size to
   1472 octets for IPv4, and 1452 octets for IPv6.
 
+- [DefragmentingDNS] concluded there is no single "magical" MTU message
+  size for DNS resolver implementations, but pointed out some safe values
+  for stub resolvers at 1400 for IPv4 and 1280 for IPv6.
```

Comment:
Above it says "DNS/UDP payload is 1400" and here is says "MTU 1400" (if not mistaken --getting late here-- MTU is for IP header and all the rest, i.e., a difference of 20 octets

Reply:
That is correct. Some references talk about the EDNS0 buffer size and some talk about the final MTU size. I did not want to modify the references with an interpretation.